model/message/validation/validators.py

```diff
@@ -23,14 +22,6 @@ def validate_run_number(run_number):
     return False
 
 
-def validate_instrument(instrument):
-    """
-    Assert an instrument exists in the valid set of instruments
-    :param instrument: The instrument to validate
-    """
-    return isinstance(instrument, str) and instrument in VALID_INSTRUMENTS
-
-
 def validate_rb_number(rb_number):
     """
     Detects if the RB number is valid e.g. (above 0 and not a string)
```

Comment:
```suggestion
    Detects if the RB number takes the format RBxxxxxxx where xxxxxxx is the 7-digit RB number
```

Reply:
I agree the original is now wrong, but I think the suggestion is a bit confusing. Maybe `Detects whether the RB number is a 7-digit number`?